contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,130 @@
+# See readme.md for instructions on running this code.
+from github import GithubConn
+import requests
+import json
+
+
+class GitHubHandler(object):
+    '''
+    This plugin facilitates allows you to comment on a GitHub issue, under a certain repository.  It
+    looks for messages starting with '@comment' or '@github-comment.
+    '''
```

Comment:
I think 'facilitates' is superfluous here. Could you also make sure the lines are short, i.e. 80 characters? And add a missing `'` before the last full stop.